demo/server/src/app.ts

```diff
@@ -14,6 +14,16 @@ import appConfig from './util/config';
 
 const app: express.Express = express();
 const config = appConfig.getConfig();
+let corsConfig = {};
+if (!config.keycloakConfig) {
+  const dynamicOrigin = function(origin, callback) {
```

Comment:
Looks like we disabling cors here by allowing all origins. Do we need to set cors then?